pkg/chunk/cassandra/storage_client.go

```diff
@@ -30,6 +30,9 @@ type Config struct {
 	Password                 string        `yaml:"password,omitempty"`
 	Timeout                  time.Duration `yaml:"timeout,omitempty"`
 	ConnectTimeout           time.Duration `yaml:"connect_timeout,omitempty"`
+	Retries                  int           `yaml:"retry_count"`
```

Comment:
`retry_count` > `max_retries` may be more clear and consistent across the config

Reply:
good callout will adjust it.